### ISSUE TITLE
"Could be uploaded by the authors" has false positives (i.e. attempts to guilt authors into things they can't do)

### ISSUE THREAD

**Issue by episanty** [dissemin/dissemin#215]
Here is one specific example:

http://dissem.in/paper/104503/

The paper in question is an editorial (News & Views) piece, for which the publisher retains the copyright (http://www.nature.com/authors/policies/license.html), and none of the self-archiving provisions hold. (Similarly, NPG retains copyright of commissioned review papers, and those should definitely be listed in Dissemin.)

The way this comes about is fairly reasonable: the paper was listed in some data source - probably CrossRef - and assigned the publisher policies set by SHERPA/Romeo [here](http://www.sherpa.ac.uk/romeo/search.php?issn=1749-4885).

As it happens, this chain of reasoning is incorrect, and it breaks when assigning the paper the blanket SHERPA/Romeo policies. As far as that page goes, SHERPA/Romeo is very careful in stating that the information is for guidance purposes, making no guarantee that it is the case for all papers in the journal.

> This summary is for the journal's default policies, and changes or exceptions can often be negotiated by authors.
> All information is correct to the best of our knowledge but should not be relied upon for legal advice.

Dissemin, on the other hand, disregards this qualifier, and assumes that anything that SHERPA/Romeo says about the uploadability of the paper is true. Dissemin then goes on to try and guilt the authors into uploading their papers based on this information.

I think this is an issue. If Dissemin wants to go above and beyond the box provided by SHERPA/Romeo around their information, then Dissemin also has a responsibility to do a good-faith attempt to find the standard cases where its assumptions fail. The way things stand now, a well-meaning author that wants their Dissemin record to say "hey, I'm all up to date with making my stuff openly available" has no way to do so.

Coding-wise, I understand this is a nightmare, and this would have to be done on a per-publisher basis, using criteria which will be hard to define and expensive to implement. However, it is still something that ought to be implemented if the current language is to be kept as it is.

As an alternative, though, Dissemin could open up ways for authors to re-classify papers from "Could be shared by the authors" to "Publisher forbids sharing", ideally with a custom note explaining the situation. With such a tool in place, and given enough such submissions, a few standard categories should be enough to classify most of the reasons that papers don't fit in the normal scheme.

Bottom line, though: by all means try and guilt people into doing the things they can and ought to do. But stop at that line and don't tell me I should be doing stuff I can't.

**Comment by wetneb** [comment]
Thank you very much for this feedback. We are aware that our classification is not always exact, and we do acknowledge that by displaying where the data comes from. Our terms of service are short and clearly indicate that the platform is not error-free: http://dissem.in/tos#scope
We could change the wording of that, or add a sentence on every paper page, or policy page, though.

Improving the heuristics is always possible. But improved heuristics remain heuristics, so they are bound to fail on some cases. Incorporating user feebdack is complicated. We are all volunteers and welcome contributions though.

**Comment by wetneb** [comment]
One thing we should do would be to add a warning when we are using a blanket policy, and not a blanket tied to a particular journal, which is the case in your paper.

**Comment by episanty** [comment]
I guess the problem is that you're always using a blanket policy. The ToS do represent that, but the wording on result pages doesn't, and expecting casual visitors to read your ToS is not realistic. SHERPA/Romeo gets it much closer, with an appropriate disclaimer on every result page. 

I think the solution is to provide a manual reclassification mechanism. This is probably a lot of building but keep it in mind, maybe for the future. Your heuristics won't always be right - so maybe the best way is to allow the user to fix them, or at least flag them, when they get stuff wrong.
